src/views/Home/Home.js

```diff
@@ -1,18 +1,34 @@
-import React from 'react'
-
+import React, { useState } from 'react'
 import Preview from '../../components/Preview/Preview'
 import Editor from '../../components/Editor/Editor'
-
 import './Home.css'
 
 export default function Home() {
   // add useState calls here for title, subtitle, font, align, and text
 
+  const [title, setTitle] = useState('')
+  const [subtitle, setSubtitle] = useState('')
+  const [text, setText] = useState('')
+  const [font, setFont] = useState('')
```

Comment:
You can also set defaults for these so they are set to something from the beginning